src/components/modals/addprojects-modal.js

```diff
@@ -24,7 +24,7 @@ export class AddProjectsModal {
   getData() {
     return this.dataContext.getRepositories(null)
       .then(projects => {
-          this.projects = JSON.parse(JSON.stringify(projects));
+          // this.projects = JSON.parse(JSON.stringify(projects));
```

Comment:
Would be helpful to note why this is commented out, or remove it.

Reply:
good catch... fixed!